Makefile

```diff
@@ -65,22 +68,14 @@ assemble-phone-release:
 	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME)  assemblePhoneRelease)
 
 assemble-app:
-	(./code/gradlew -p code/$(TEST-APP-FOLDER-NAME) assemble)
+	(./code/gradlew -p code/$(TEST-APP-FOLDER-NAME-JAVA) assemble)
+	(./code/gradlew -p code/$(TEST-APP-FOLDER-NAME-KOTLIN) assemble)
 
-ci-publish: clean assemble-phone-release
-	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) publishReleasePublicationToSonatypeRepository -Prelease)
+ci-publish-maven-local-jitpack: assemble-phone-release
+	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) publishReleasePublicationToMavenLocal -Pjitpack  -x signReleasePublication)
 
 ci-publish-staging: clean assemble-phone-release
 	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) publishReleasePublicationToSonatypeRepository)
 
-# usage: update-version VERSION=9.9.9 CORE-VERSION=8.8.8
-# usage: update-version VERSION=9.9.9
-update-version:
-	@echo "Updating version to $(VERSION), Core version to $(CORE-VERSION)"
-	sed -i '' "s/[0-9]*\.[0-9]*\.[0-9]/$(VERSION)/g" ./code/edge/src/main/java/com/adobe/marketing/mobile/EdgeConstants.java
-	sed -i '' "s/\(moduleVersion=\)[0-9]*\.[0-9]*\.[0-9]/\1$(VERSION)/g" ./code/gradle.properties
-	@if [ -z "$(CORE-VERSION)" ]; then \
-		echo "CORE-VERSION was not provided, skipping"; \
-	else \
-		sed -i '' "s/\(mavenCoreVersion=\)[0-9]*\.[0-9]*\.[0-9]/\1$(CORE-VERSION)/g" ./code/gradle.properties; \
-	fi
+ci-publish: clean assemble-phone-release
+	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) publishReleasePublicationToSonatypeRepository -Prelease)
```

Comment:
we can remove `assemble-phone-release` for `-publish-staging` and `ci-publish`, they are taken care by `aep-library`.
refer to https://github.com/adobe/aepsdk-edgeconsent-android/pull/77#discussion_r1483768071